code/index.html

```diff
@@ -1,22 +1,81 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Gotland news</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+
+    <!-- link for icon -->
+    <script src="https://kit.fontawesome.com/07170c8548.js" crossorigin="anonymous"></script>
+
+    <!-- meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- link to font-family -->
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
   </head>
+
   <body>
+    <a name="top"></a>
+    
+    <!-- Header nav bar-->
     <header>
-      <!-- Some kind of logo and navbar -->
+      <nav class="navigator">
+        <i class="far fa-sun"></i>
```

Comment:
I can see that you have named the classes as the icon name!
On top of my mind, it might be a good habit to name the classes more generally so you can re-use the code for other purposes next time. For example maybe you can name them 'navbar left' 'navbar logo' 'navbar right' so you can directly see which classes are they for. What do you think? :)